doc/en/user/source/datadirectory/setting.rst

```diff
@@ -3,14 +3,21 @@
 Setting the data directory location
 ===================================
 
-The procedure for setting the location of the GeoServer data directory is dependent on the type of GeoServer installation. Follow the instructions below specific to the target platform. 
+The ``GEOSERVER_DATA_DIR`` application property are determined using the first value obtained from: Java System Properties, Web Application context parameters, or System Environmental Variable.
+
+* Follow the instructions below to learn how to set the environmental variable location. There are different instructions for each target platform.
+
+* The page on :ref:`application_properties` has instructions for defining ``GEOSERVER_DATA_DIR`` using a Java System Property or Web Application context parameters.
 
 .. note:: If the location of the GeoServer data directory is not set explicitly, the directory ``data_dir`` under the root of the GeoServer installation will be chosen by default.
 
+
 Windows
 -------
 
-On Windows platforms the location of the GeoServer data directory is controlled by the ``GEOSERVER_DATA_DIR`` environment variable.
+On Windows platforms the location of the GeoServer data directory is controlled by the ``GEOSERVER_DATA_DIR`` environment variable, environment variable, system property, or web context parameter.
+
+* :file:`C:\\ProgramData\\GeoServer` (example location)
```

Comment:
```suggestion
* :file:`%PROGRAMDATA%\\GeoServer` (example location)
```

Reply:
Should we do both examples? Or is `%PROGRAMDATA%` understood enough ...

Yes it is a long time since I even had a windows VM.